### PR TITLE
Add a note that `+ssc` schemes are not supported

### PR DESCRIPTION
**Pull request by renetapopova**
Depends on https://github.com/neo-technology/neo4j-manual-modeling/pull/2774, which should be merged first.

### CODE REVIEW

File: docs/modules/ROOT/pages/visual-tour.adoc [hunk 1]
```diff
@@ -198,7 +198,7 @@ To access the list of all available editor keybindings, press F1 in the Cypher e
 You can click a block shown in the history result to bring it into the editor.
 The command history is persisted across Browser restarts.
 
-| `CALL db.schema.visualization()`
+| `CALL db..visualization()`
```
Comment (gregoryking) [inline docs/modules/ROOT/pages/visual-tour.adoc:201]: Is this a find and replace editing error?

**Review by gregoryking — CHANGES_REQUESTED** [review]
Looks good overall, just noticed one file possibly has a mistaken edit in it.

File: docs/modules/ROOT/pages/visual-tour.adoc (hunk 1 above)
Comment (renetapopova) [reply docs/modules/ROOT/pages/visual-tour.adoc:201]: Of course, good spot!!

Comment (renetapopova) [reply docs/modules/ROOT/pages/visual-tour.adoc:201]: Aah, yes, it is a replace error. Thanks!

Comment (renetapopova) [inline docs/modules/ROOT/pages/visual-tour.adoc:201]:
```suggestion
| `CALL db.schema.visualization()`
```

**Pull request merged**